x-pack/test/api_integration/apis/ml/modules/setup_module.ts

```diff
@@ -311,33 +311,6 @@ export default ({ getService }: FtrProviderContext) => {
         dashboards: [] as string[],
       },
     },
-    {
-      testTitleSuffix:
```

Comment:
Noting that removal of these tests leaves us with no CI coverage inside ML (API integration or functional tests) for the creation of the `security_linux` and `security_windows` jobs. The ML data team are looking at creating some new downsampled data sets which we can use in CI, and we will aim to add back some tests in a follow-up, so no need to hold up this PR.